mod-finance/schemas/fy_finance_data.json

```diff
@@ -85,6 +85,10 @@
       "description": "The current allocation amount for this budget",
       "type": "number"
     },
```

Comment:
budgetInitialAllocation and budgetCurrentAllocation seems duplicated fields
In UI we have only single field for this
![image](https://github.com/user-attachments/assets/42ac2a93-3d07-4017-8499-401b1e918df9)